heron/stmgr/src/cpp/manager/stmgr-server.cpp

```diff
@@ -30,37 +30,61 @@
 #include "network/network.h"
 #include "config/helper.h"
 #include "config/heron-internals-config-reader.h"
+#include "metrics/metrics.h"
 
 namespace heron {
 namespace stmgr {
 
 // Num data tuples received from other stream managers
```

Comment:
Please add a comment about why we have to register these metrics one by one instead of registering the scope.